src/isIsogram.test.js

```diff
@@ -6,4 +6,18 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it('should return true if comes an empty string', () => {
+    expect(isIsogram('')).toBeTruthy();
+  });
+
+  it('should return correct value if a non-empty string is received', () => {
+    expect(isIsogram('playgrounds')).toBeTruthy();
+    expect(isIsogram('look')).toBeFalsy();
```

Comment:
Write each 'expect' in a separate 'it', so that you check only one situation in one test
You test different scenarios: please, separate them into different and provide a more specific explanation which input you are checking